lib/auth/acl.js

```diff
@@ -0,0 +1,79 @@
+'use strict';
+
+/**
+ * @fileoverview Access Control Lists
+ */
+const config = require('config');
+const NodeAcl = require('acl');
+
+const userRoles = config.roles.config;
+
+// memoryBackend starts with a lowercase from node-acl
+const acl = new NodeAcl(new NodeAcl.memoryBackend()); // eslint-disable-line new-cap
```

Comment:
So, found your problem with everyone being unloaded on server crash. Using memory backend. Should use [knex backend](https://github.com/christophertrudel/node_acl_knex)